src/D3MHub.sol

```diff
@@ -395,128 +395,47 @@ contract D3MHub {
         @param ilk bytes32 of the D3M ilk name
     */
     function exec(bytes32 ilk) external lock {
+        // IMPORTANT: this function assumes Vat rate of D3M ilks will always be == 1 * RAY (no fees).
+        // That's why this module converts normalized debt (art) to Vat DAI generated with a simple RAY multiplication or division
+
         (uint256 Art, uint256 rate, uint256 spot, uint256 line,) = vat.ilks(ilk);
         require(rate == RAY, "D3MHub/rate-not-one");
         require(spot == RAY, "D3MHub/spot-not-one");
 
         ID3MPool _pool = ilks[ilk].pool;
 
         _pool.preDebtChange("exec");
```

Comment:
Not a major thing, but since we removed `reap` we should consider removing the "exec" identifier in `pool.preDebtChange()` and `pool.postDebtChange()`, as these are callend only in `exec()` now.

Reply:
Good catch, it's done now!